src/components/TextLink.js

```diff
@@ -22,20 +22,22 @@ const propTypes = {
 
     /** Overwrites the default link behavior with a custom callback */
     onPress: PropTypes.func,
+    onMouseDown: PropTypes.func,
```

Comment:
We should add comment for the `onMouseDown` prop like _**Callback that is called when mousedown is triggered.**_